### PR TITLE
Dawids branch

### PR DESCRIPTION
**Pull request by 77masterdawid77**
Added alot of assets we can use

### CODE REVIEW

**Review by Shinkson47 — CHANGES_REQUESTED** [review]
This commit should not be included. It was squashed in c256b9f, thus It's content already exists on the master.

Instead of branching at the and of master, you based your branch at the beginning, the very first commit before anything was added, then merged master into your branch.

Capture d’écran 2021-05-23 à 1 46 58 PM


I'll cherry pick to a new branch to clean it up.

**Review by Shinkson47 — COMMENTED** [review]
Assets themselves look good, but half of them are unexported photoshop files which i believe gdx can't load. Should we include them anyway?

**Comment by 77masterdawid77** [comment]
don't include the photoshop files, Ill change them to .png later, thanks.

File: core/src/com/dmugang/JavaProject.java [hunk 1]
```diff
@@ -7,25 +7,29 @@
...
 	}
 
 	@Override
 	public void render () {
-		ScreenUtils.clear(1, 0, 0, 1);
+		ScreenUtils.clear(0, 0, 1, 1);
 		batch.begin();
-		batch.draw(img, 0, 0);
+		batch.draw(playerSprite, 250, 100);
+		batch.draw(backgroundSprite, 50, 100);
```
Comment (Shinkson47) [inline core/src/com/dmugang/JavaProject.java:25]: Last thing, I promise.

Now that #2 is merged, we won't be doing anything like this directly in the client class.

Game rendering can be done in a *game screen* class, and set to the client as mentioned in #2.

It's a conflict that will have to be resolved before we can merge

Comment (77masterdawid77) [reply core/src/com/dmugang/JavaProject.java:25]: ok, if you need just delete those things I have done and Ill add them later correctly. We just might need to do a meeting for someone to explain how this works for me

Comment (Shinkson47) [reply core/src/com/dmugang/JavaProject.java:25]: I'll take a couple minutes to merge them together, but i can walk you through it if you'd like.

**Pull request closed without merging**